src/common/util.js

```diff
@@ -429,7 +429,7 @@ function setupi18nTranslation(dict) {
       }
 
       function localize(node) {
-          if (node && node.className && node.className.includes('no-translation')) return;
+          if (node && node.className !== undefined && node.className.includes('no-translation')) return;
```

Comment:
can we just check `node?.className` here

Reply:
I have tried node?.className but because of version issue so it couldn't be accepted when i try to build it.